joebvp/atomicdata/__init__.py

```diff
@@ -17,6 +23,14 @@
 vernosc=jbg.arrfromcol(vernerlist,6)
 vernp=jbg.arrfromcol(vernerlist,7)
 
+starts = (0,10,18,21,24,27,30,40)
+verntab = Table.read(jbvp_path+'/atomicdata/verner6.txt',
+                     format='ascii.fixed_width_no_header',
+                     col_starts=starts,fill_values=('','0'),
+                     names=['lam','ion','Z','nume','gl','gu','fosc','P'])
+verntab['fosc'].astype(float)
+vernion=verntab['ion']
+
```

Comment:
I think Verner table is included in LineList 'ISM'

Reply:
True, and I just verified that linetools is being used for all instances where we are retrieving actual atomic data.  The one thing we still look up in the Verner table is the species name, e.g., 'H I' where the LineList class doesn't simply serve up the species name.  I'm inclined to deprecate this at some point and add a 'species' attribute to LineList.  Gonna punt for now though.

Reply:
Do not follow, you can get the species names in LineList object by removing the second part of the name string...

Reply:
This is true, but the spacing between, e.g., 'H I' and 'SiIII' as they appear in our output tables.